src/include/pbs_ifl.h

```diff
@@ -85,31 +85,32 @@ extern "C" {
 #define ATTR_N "Job_Name"
 #define ATTR_S "Shell_Path_List"
 #define ATTR_array_indices_submitted ATTR_J
-#define ATTR_depend		"depend"
-#define ATTR_inter		"interactive"
-#define ATTR_sandbox		"sandbox"
-#define ATTR_stagein		"stagein"
-#define ATTR_stageout		"stageout"
-#define ATTR_resvTag		"reserve_Tag"
-#define ATTR_resv_start		"reserve_start"
-#define ATTR_resv_start_revert	"reserve_start_revert"
-#define ATTR_resv_end		"reserve_end"
-#define ATTR_resv_duration	"reserve_duration"
-#define ATTR_resv_state		"reserve_state"
-#define ATTR_resv_substate	"reserve_substate"
-#define ATTR_resv_job		"reserve_job"
-#define ATTR_auth_u		"Authorized_Users"
-#define ATTR_auth_g		"Authorized_Groups"
-#define ATTR_auth_h		"Authorized_Hosts"
-#define ATTR_cred		"cred"
-#define ATTR_nodemux		"no_stdio_sockets"
-#define ATTR_umask		"umask"
-#define ATTR_block		"block"
-#define ATTR_convert            "qmove"
-#define ATTR_DefaultChunk	"default_chunk"
-#define ATTR_X11_cookie         "forward_x11_cookie"
-#define ATTR_X11_port           "forward_x11_port"
-#define ATTR_GUI		"gui"
+#define ATTR_depend			"depend"
+#define ATTR_inter			"interactive"
+#define ATTR_sandbox			"sandbox"
+#define ATTR_stagein			"stagein"
+#define ATTR_stageout			"stageout"
+#define ATTR_resvTag			"reserve_Tag"
+#define ATTR_resv_start			"reserve_start"
+#define ATTR_resv_start_revert		"reserve_start_revert"
+#define ATTR_resv_end			"reserve_end"
+#define ATTR_resv_duration		"reserve_duration"
+#define ATTR_resv_duration_revert	"reserve_duration_revert"
+#define ATTR_resv_state			"reserve_state"
+#define ATTR_resv_substate		"reserve_substate"
+#define ATTR_resv_job			"reserve_job"
+#define ATTR_auth_u			"Authorized_Users"
+#define ATTR_auth_g			"Authorized_Groups"
+#define ATTR_auth_h			"Authorized_Hosts"
+#define ATTR_cred			"cred"
+#define ATTR_nodemux			"no_stdio_sockets"
+#define ATTR_umask			"umask"
+#define ATTR_block			"block"
+#define ATTR_convert			"qmove"
+#define ATTR_DefaultChunk		"default_chunk"
+#define ATTR_X11_cookie			"forward_x11_cookie"
+#define ATTR_X11_port			"forward_x11_port"
+#define ATTR_GUI			"gui"
```

Comment:
If you are going  to format these to line them up, some of them are further indented than others.

Reply:
Oh no,
It shows up perfectly in my editor
<img width="554" alt="Screen Shot 2020-07-23 at 9 59 18 AM" src="https://user-images.githubusercontent.com/17924134/88315576-4d910b00-cccb-11ea-8860-a5e312ee0c5d.png">